template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Startup.cs

```diff
@@ -47,14 +55,28 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
 #if (IndividualLocalAuth)
-            services.AddIdentityServiceAuthentication();
+            services.AddDbContext<IdentityServiceDbContext>(options =>
+                options.UseSqlServer(Configuration.GetConnectionString("DefaultConnection")));
+
+            services.AddIdentity<ApplicationUser, IdentityRole>()
+                .AddDefaultTokenProviders()
+                .AddApplications()
+                .AddEntityFrameworkStores<IdentityServiceDbContext>()
+                .AddClientExtensions();
+
 #elseif (IndividualB2CAuth)
             services.AddAzureAdB2CAuthentication();
 #elseif (OrganizationalAuth)
             services.AddAzureAdAuthentication();
 #endif
```

Comment:
Do we want to use the commented out conditional syntax for some of this, so the template source would be runnable? (same would apply to other places)